cmd/flow-rosetta-server/main.go

```diff
@@ -63,15 +64,15 @@ func run() int {
 	// Command line parameter initialization.
 	var (
 		flagDPSAPI       string
-		flagFlowAPI      string
+		flagAccessAPI    string
 		flagCache        uint64
 		flagLevel        string
 		flagPort         uint16
 		flagTransactions uint
 	)
 
-	pflag.StringVarP(&flagDPSAPI, "api", "a", "127.0.0.1:5005", "host URL for GRPC API endpoint")
-	pflag.StringVarP(&flagFlowAPI, "flow-api", "f", "", "host URL for Flow network's Access API endpoint")
+	pflag.StringVarP(&flagDPSAPI, "dps-api", "a", "127.0.0.1:5005", "host URL for GRPC API endpoint")
+	pflag.StringVarP(&flagAccessAPI, "access-api", "f", "", "host URL for Flow network's Access API endpoint")
```

Comment:
Do we want to keep `f` as the shorthand flag for this arg after renaming it? I'd be fine with it since `a` is already taken, but asking just in case.

Reply:
Yeah, the shorthand `f` doesn't make sense anymore. I don't know, perhaps it makes sense the remove the shorthand altogether.